stock_request/data/stock_request_sequence_data.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
remove data tag

Reply:
Not in this case: *Remove \<data\> and \</data\> in xml files **if noupdate="0"***

Reply:
in v11 should be put as `<odoo noupdate="1">`